tests/tests/in-cluster-eventing.spec.js

```diff
@@ -121,17 +119,30 @@ context('In-cluster eventing', () => {
       .click();
 
     cy.getIframeBody()
-      .contains('a', FUNCTION_NAME)
+      .contains('a', API_RULE_AND_FUNCTION_NAME)
       .click();
 
     cy.getIframeBody()
-      .contains('View Logs')
-      .click();
-
-    // it just doesn't work in cypress
-    // cy.getIframeBody()
-    //   .contains('.logs', 'Event received')
-    //   .should('be.visible');
+      .contains(`${API_RULE_AND_FUNCTION_NAME}-`)
+      .then(element => {
+        const podName = element[0].textContent;
+        loadKubeconfig().then(kubeconfig => {
+          const requestUrl = `/api/v1/namespaces/${Cypress.env(
+            'NAMESPACE_NAME',
+          )}/pods/${podName}/log?container=function`;
+          cy.request({
+            method: 'GET',
+            url: kubeconfig.clusters[0].cluster.server + requestUrl,
+            timeout: 10000,
+            headers: {
+              authorization: 'Bearer ' + kubeconfig.users[0].user.token,
+            },
+          }).then(response => {
+            // response.body is automatically serialized into JSON
+            expect(response.body).to.match(/^Payload/);
```

Comment:
Shouldn't we check if we received "Event received" log entry?

Reply:
No, we test eventing for publisher function, not for receiver.

Reply:
We have ready `API Rule` for the publisher so that's why I'm testing publisher, not reciver.

Reply:
But we should test if receiver received the event.

Reply:
In the end, we test both.